client/app/components/authors/header/menu/AuthorMenu.jsx

```diff
@@ -19,48 +19,63 @@ const AuthorMenu = ({
         return `pages-menu--mobile ${isMobileMenuOpen ? "pages-menu--mobile-visible" : ""}`;
     };
 
+    const hasCredentials = () => author.credentials.length > 0;
+
+    const hasPages = () => pages && pages.length > 0;
+
+    const hasGuestbook = () => !author.guestbook_disabled;
+
+    const hasNewsletter = () => !author.newsletter_disabled;
+
+    const shouldRenderMenu = () => hasPages() || hasCredentials()
+        || hasGuestbook() || hasNewsletter();
```

Comment:
```suggestion
    const hasCredentials = author.credentials.length > 0;

    const hasPages = pages && pages.length > 0;

    const hasGuestbook = !author.guestbook_disabled;

    const hasNewsletter = !author.newsletter_disabled;

    const shouldRenderMenu = hasPages || hasCredentials
        || hasGuestbook || hasNewsletter;
```

I think these can all just be constant conditions instead of functions 🙂

Reply:
I'm moving this to another file and sending the `author` as a parameter, so it will be a function 😅 
But I'll take that into account for next time! Thank you @antsgar